force-app/main/default/classes/searchAndPick.cls

```diff
@@ -0,0 +1,27 @@
+global with sharing class searchAndPick {
+          private final Account person;
+          public searchAndPick(ApexPages.StandardController stdController){
```

Comment:
Please fix indentation and select more descriptive class name such as PrimaryAccountExtension or similar.